pkgs/os-specific/linux/kernel/patches.nix

```diff
@@ -60,4 +60,14 @@
     name = "export-rt-sched-migrate";
     patch = ./export-rt-sched-migrate.patch;
   };
+
+  # Backport upstream fix for very annoying Tailscale issue
+  # FIXME: remove when merged upstream
+  netfilter-typo-fix = {
+    name = "netfilter-typo-fix";
+    patch = fetchpatch {
+      url = "https://git.kernel.org/pub/scm/linux/kernel/git/netdev/net.git/patch/?id=306ed1728e8438caed30332e1ab46b28c25fe3d8";
```

Comment:
Do we know this branch doesn't get rebased? Otherwise would be better to fetch from lore.

Reply:
I don't think net-next gets rebased much, but even if it is, we should hopefully only have the patch for a week...

Reply:
But in the history forever, and the kernel is quite likely to be something needing bisected!

But it seems to have multiple committers, so that's a good sign.